.circleci/config.yml

```diff
@@ -810,90 +810,8 @@ workflows:
   #         context: Slack Orb
   main-workflow:
       jobs:
-        - build-job:
-            name: "Xcode_13.2.1_iOS_15.2_Debug"
-            xcode: "13.2.1"
-            iOS: "15.2"
-            device: "iPhone 13 Pro Max"
-            context: Slack Orb
-        - build-job:
-            name: "Xcode_13.2.1_iOS_15.2_SPM_Debug"
-            xcode: "13.2.1"
-            iOS: "15.2"
-            device: "iPhone 13 Pro Max"
-            spm: true
-            codecoverage: false
-            context: Slack Orb
-        - build-job:
-            name: "Xcode_13.2.1_iOS_15.2_Release"
-            xcode: "13.2.1"
-            iOS: "15.2"
-            device: "iPhone 13 Pro Max"
-            context: Slack Orb
-            configuration: "Release"
-        - build-job:
-            name: "Xcode_13.2.1_iOS_15.2_SPM_Release"
-            xcode: "13.2.1"
-            iOS: "15.2"
-            device: "iPhone 13 Pro Max"
-            spm: true
-            codecoverage: false
-            context: Slack Orb
-            configuration: "Release"
-        - build-job:
-            name: "Xcode_13.4.1_iOS_13.7"
-            xcode: "13.4.1"
-            iOS: "13.7"
-            device: "iPhone 11 Pro Max"
-            spm: true
-            codecoverage: false
-            context: Slack Orb
-        - build-job:
-            name: "Xcode_14.0_iOS_16.0"
-            xcode: "14.0.0"
-            iOS: "16.0"
-            device: "iPhone 14 Pro Max"
-            spm: true
-            codecoverage: false
-            context: Slack Orb
-        - pod-job:
-            name: "Xcode_13.2.1_iOS_15.2_CP_install"
-            update: false
-            xcode: "13.2.1"
-            iOS: "15.2"
-            archive: true
-            context: Slack Orb
-        - pod-job:
-            name: "Xcode_13.2.1_iOS_15.2_CP_update"
-            update: true
-            xcode: "13.2.1"
-            iOS: "15.2"
-            lint: true
-            context: Slack Orb
-        - build-example:
-            name: "Build Example"
-            context: Slack Orb
-        - spm-core-integration-test-job:
-            name: "Xcode 13; iOS 15.0; SPM Core test"
-        - build-job:
-            name: "Core integration test; Xcode 14; iOS 16.0"
-            xcode: "14.0.0"
-            iOS: "16.0"
-            device: "iPhone 14 Pro Max"
-            scheme: "MapboxCoreNavigationIntegrationTests"
-            codecoverage: false
-            clean_build: false
-            context: Slack Orb
-        - generate-docs-job:
-            name: Generate / Publish Documentation
-            context: Slack Orb
-            filters:
-              tags:
-                only: /^v\d+\.\d+\.\d+(-.+)?$/
-        - ios-trigger-metrics:
-            filters:
-              branches:
-                only: main
```

Comment:
Are really all those jobs should be removed?

Reply:
PR is not ready for review yet. Accidentally removed draft label.